### ISSUE TITLE
Add Privacy Text on SDK installation Site

### ISSUE THREAD

**Issue by JekCharlsonYu** [dart-lang/site-www#968]
Can we put a banner or text under the Install SDK site some information about our privacy policy?
https://webdev-dartlang-org-dev.firebaseapp.com/tools/sdk#install

Will send a sample text over email with Smithya.

**Comment by kwalrath** [comment]
Every page on the site has a link to Privacy terms at the bottom. Do we need more?

**Comment by JekCharlsonYu** [comment]
Yup, they are asking for a big text around where you can download the sdk
so it is clear.

I have cc-ed you on the separate email
